package/Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG UBI_IMAGE=registry.access.redhat.com/ubi8/ubi-minimal:latest
+FROM ${UBI_IMAGE}
+ENV KUBECONFIG /root/.kube/config
```

Comment:
What is the point of this kubeconfig, does that get dynamically set later on ?

Reply:
There is a command line flag that uses the environment variable for connecting to a different Kubernetes cluster.